src/plugin.js

```diff
@@ -10,18 +10,29 @@ const notUrl = (url) => url.substr(0, 4) !== 'http';
 const notBare = (str) =>
     str.startsWith('/') || str.startsWith('./') || str.startsWith('../');
 
-async function readEikJSONMaps(eikJSONPath) {
+async function readJSONFile(path) {
     try {
-        const contents = await fs.promises.readFile(eikJSONPath);
-        const eikJSON = JSON.parse(contents);
-        if (typeof eikJSON['import-map'] === 'string')
-            return [eikJSON['import-map']];
-        return eikJSON['import-map'] || [];
+        const contents = await fs.promises.readFile(path);
+        return JSON.parse(contents);
     } catch (err) {
-        return [];
+        return {};
     }
 }
 
+async function readEikJSONMaps(eikJSONPath, pkgJSONPath) {
+    const eikJSON = await readJSONFile(eikJSONPath);
+    const pkgJSON = await readJSONFile(pkgJSONPath);
```

Comment:
We're using promises here to be non blocking but there is no real benefit to this due to each step being done in a sync order.

If we make `readJSONFile` into returning a promise which resolve with a object or rejects, these two lines can be run in an awaited `Promise.all()` and both files will be read in parallel. Which should improve speed.